pkg/sql/backfill/backfill.go

```diff
@@ -491,7 +652,7 @@ func (ib *IndexBackfiller) BuildIndexEntriesChunk(
 		if buffer, err = sqlbase.EncodeSecondaryIndexes(
 			ib.evalCtx.Codec,
 			tableDesc.TableDesc(),
-			ib.added,
+			ib.indexesToEncode,
```

Comment:
oh my goodness its because this takes a slice of nonpointers. That's unfortunate. I'm not sure whats the best thing to do here. I feel like these copies are going to show up in a profile for sure.

Reply:
I'm going to try to change this to take a slice of pointers. Maybe it won't be so bad...

Reply:
I think it might be bad :D We use this mainly when we have a table descriptor, so we take that slice of indexes right off of there.